download/download.go

```diff
@@ -154,6 +159,7 @@ func Download(args []string) error {
 		urlsFilePath = urls[0]
 	}
 
+	log.Print(urlsFilePath)
```

Comment:
Just checking that this log is supposed to be here, and not a forgotten debug log? ^^